datahub-web-react/src/app/entity/dataset/DatasetEntity.tsx

```diff
@@ -252,6 +264,7 @@ export class DatasetEntity implements Entity<Dataset> {
                 platformLogo={data.platform.properties?.logoUrl}
                 platformInstanceId={data.dataPlatformInstance?.instanceId}
                 owners={data.ownership?.owners}
+                access={this.isAccessManagementEnabled() ? data?.access : null}
```

Comment:
Please do not change this preview DatasetEntity.tsx - what are you hoping to achieve with this?

Reply:
done update